recipe_scrapers/ahealthysliceoflife.py

```diff
@@ -0,0 +1,36 @@
+from ._abstract import AbstractScraper
+from ._utils import normalize_string
+
+
+class AHealthySliceOfLife(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "ahealthysliceoflife.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredients = self.soup.find("div", {"class": "tasty-recipes-ingredients-body"}).findAll("p")
```

Comment:
```suggestion
        ingredients = self.soup.find(
            "div", {"class": "tasty-recipes-ingredients-body"}
        ).findAll("p")
```

> I'm using [jj](https://github.com/martinvonz/jj) instead of Git and that appears to break the pre-commit script, any way to manually run the linters?

It looks like jj is not current compatible with pre-commit hooks, but there is an open issue [here](https://github.com/martinvonz/jj/issues/405)

This particular [pre-commit](https://github.com/hhursev/recipe-scrapers/blob/main/.pre-commit-config.yaml) issue is formatting-related (see the suggestion). You could install [black](https://black.readthedocs.io/en/stable/getting_started.html) directly and run:
```python
black .\recipe_scrapers\ahealthysliceoflife.py
```

to apply the library formatting to your file.